factory/api/apiResolverFactory.go

```diff
@@ -407,25 +417,56 @@ func createScQueryElement(
 	}
 
 	var apiBlockchain data.ChainHandler
+	var accAdapter state.AccountsAdapterAPI
```

Comment:
I think these declarations can be removed.
We only need below to assign in `argsHook` the correct pointers